source/index.html.md

```diff
@@ -82,3 +83,60 @@ There are also contextual links to other related content resources contained wit
           ...
 ```
 
+The response also contains the content body:
+
+```
+  "body" : "\n<div class=\"highlight-answer\">\n<p>The standard <abbr title=\"Value Added Tax\">VAT</abbr> rate is <em>20%</em></p>\n</div>..."
+```
+
+#### Format specific response fields.
+
+Different styles of content on GOV.UK are displayed in distinct _formats_.
+
+For example, simple guidance on current VAT rates can be succinctly displayed in a single page format.
+Whereas [The Highway Code ](https://www.gov.uk/guidance/the-highway-code) is ordered into multiple sections across several pages.
+
+The GOV.UK Content API exposes these format specific differences in a discrete field called `details`.
+
+In the example above the `body` field containing the content body is embedded in the `details` field.
+
+Compare this to the `details` field from the API response for The Highway Code:
+
+```
+curl -X GET https://www.gov.uk/api/content/guidance/the-highway-code -H 'Accept: application/json' | json_pp
+```
+
+```
```

Comment:
I think we might have to set the syntax to `json` explicitly, because GitHub is detecting this as HTML.